### PR TITLE
My branch

### PR DESCRIPTION
**Pull request by vincentiroleh**
1. Corrected a misspelled word
2. Added Hello-bot-cloudinary (A messenger bot)

### CODE REVIEW

**Comment by codehakase** [comment]
@vincentiroleh the project doesn't fully satisfy https://github.com/acekyd/made-in-nigeria/blob/master/contributing.md

File: contributing.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ There is a webpage that has been developed by awesome members of the community u
 
 * If you are fixing a bug or adding a new feature, please create a branch for it.
 
-* Make Pull requests as decriptive as possible.
+* Make Pull requests as descriptive as possible.
```
Comment (codehakase) [inline contributing.md]: Thanks for noticing that, but this change should be ported to a different PR

Comment (vincentiroleh) [reply contributing.md]: Done
